packages/checkbox/src/CheckboxEditor.tsx

```diff
@@ -86,29 +99,41 @@ export function CheckboxEditor(props: CheckboxEditorProps) {
           setValue(newValues);
         };
 
+        const mergedOptions = getMergedOptions(values, options);
```

Comment:
Given these are all internal methods what would you think about having `getMergedOptions` return `{ options, invalid }`